### DISCUSSION TITLE
Student Document is not accepted

### DISCUSSION THREAD

**Discussion by ahmedelshkh** [community/community#118605] [GitHub Education]
### Hi everyone,

Hi,

I'm from Egypt so my Student Id is in Arabic
I upload it and it keeps rejecting me and says I must complete my profile. Just so you know, my profile is already complete.
![218281382-b1370f29-f361-4b40-9439-4ba79eeb8bde](https://github.com/community/community/assets/53626145/abd38b3c-1341-44d4-bf67-dc3a041167a1)

Is there a way for me to fill in my student id data manually? or what do I have to do?

**Comment by samus-aran** [comment]
Hi @ahmedelshkh 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I need to translate my evidence. What can I do?](https://github.com/orgs/community/discussions/111352#translate-my-evidence)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
